build/elasticsearch-alpine-base/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM alpine:latest
 # Update image and install base packages
 RUN apk update && \
     apk upgrade && \
-    apk add bash openjdk8 openssl && \
+    apk add bash openjdk8 openssl curl && \
```

Comment:
Can you please add `curl` after `bash` so that the packages are in alphabetical order? Thanks!